### PR TITLE
Create 2018-06-28-joint-narrative-for-ec.md

### PR DESCRIPTION
**Pull request by weaverbel**
DO NOT MERGE YET - awaiting review by @tracykteal - and only to appear on 28 June

### CODE REVIEW

File: _posts/2018/06/2018-06-28-joint-narrative-for-ec.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+various programs. The report complements 
+the [programmatic assessment work](https://carpentries.org/blog/2018/06/programmatic-assessment/) 
+recently compiled and published by our Program Manager, Maneesha Sane.
+
+This report summarises work from January 2017 to March 2018. It covers workshop administration, 
+assessment, instructor training, curriculum development and maintenance, community engagement and 
+volunteer management, and communications. If you want to get a broad overview of what is happening in The 
+Carpentries, this is the place to start!
+
+The report is published on Zenodo: https://zenodo.org/record/1294081#.Wyq0CBJKi_E
```
Comment (fmichonneau) [inline _posts/2018/06/2018-06-28-joint-narrative-for-ec.md]: make a link, and replace zenodo URL by DOI: https://doi.org/10.5281/zenodo.1294080

File: _posts/2018/06/2018-06-28-joint-narrative-for-ec.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Stay tuned for a report from the EC meeting. This report will appear on
+The Carpentries [blog](https://carpentries.org/blog), and a link to it will also be 
+shared via our [Twitter feed](https://twitter.com/thecarpentries), 
+our [Facebook](https://facebook.com/carpentries)page, and
```
Comment (fmichonneau) [inline _posts/2018/06/2018-06-28-joint-narrative-for-ec.md]: missing space after ) before "page"

File: _posts/2018/06/2018-06-28-joint-narrative-for-ec.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+organisation, [The Carpentries](https://carpentries.org/). In mid-June, The Carpentries held  their 
+first [Executive Council](http://static.carpentries.org/governance/) in-person meeting. 
+A report of that meeting will be published shortly. 
+
+In preparation for the EC meeting, Carpentries' staff prepared a narrative report on the progress we have made 
+as a joint organisation since we began to work together more closely from January 2017. 
+Two key milestones during that time were the creation and launch 
+of both [The Carpentries Handbook](https://docs.carpentries.org/) and the new Carpentries [website](https://carpentries.org/).
+
+See our report: https://zenodo.org/record/1294081#.Wyq0CBJKi_E
```
Comment (fmichonneau) [inline _posts/2018/06/2018-06-28-joint-narrative-for-ec.md]: same comment as below about using DOI link. Do we need to repeat the link?

**Comment by weaverbel** [comment]
Thanks @fmichonneau - everything shipshape now ? Don't want this merged though,

File: _posts/2018/06/2018-06-28-joint-narrative-for-ec.md (hunk 1 above)
Comment (weaverbel) [reply _posts/2018/06/2018-06-28-joint-narrative-for-ec.md]: Done!

**Comment by fmichonneau** [comment]
Yes, it looks good!

**Comment by weaverbel** [comment]
I don't think it hurts to have the link twice @fmichonneau

**Pull request merged**